app/controllers/articles_controller.rb

```diff
@@ -18,6 +18,7 @@ def create
     @article = Article.new(article_params)
 
     if @article.save
+      ArticleMailer.with(user: current_user).create_mail(@article).deliver_now
```

Comment:
we don't need that

Reply:
sir comment out kr du ya permanent hta du

Reply:
remove that